### DISCUSSION TITLE
Pdf text highlights in rotated format

### DISCUSSION THREAD

**Discussion by Swejal08** [wojtekmaj/react-pdf#1788] [Q&A]
I am trying to highlight text in a rotated PDF.
But I'm encountering issue where  the highlighted part does not get rotated along with the pdf .
I am using `customTextRenderer` to highlight the text. 

![image](https://github.com/wojtekmaj/react-pdf/assets/70813833/233bdd08-eb7d-4662-97f0-900a45ce9a90)

**Comment by wojtekmaj** [comment]
Do you have a repro in CodeSandbox or similar tool?
